### PR TITLE
Various sample updates

### PR DESCRIPTION
**Pull request by SatjaSivcev**
This commit:

New samples
- Adds TransformPointCloudFromMillimetersToMeters sample.
  This sample transforms   a point cloud from millimeters
  to meters using transform API.

Modifications
- Fixes a typo in GetCameraIntrinsics sample
- Adds export to ply to Capture sample
- Adds save and load settings to CaptureHDRCompleteSettings sample
- Adds downsample as a new point cloud instance in addition to the
  in-place in downsampling sample
- Updates UtilizeEyeInHand sample and renames it to UtilizeHandEye.
  This sample now works with a new dataset for eye-in-hand
  (located in SampleData.zip), consisting of:
  - RobotTransform
  - EyeInHandTransform
  - ZividGemEyeInHand
  - ZividGemEyeToHandTransformed
  and the following dataset for eye-to-hand, consisting of:
  - ZividGemEyeToHand
  - ZividGemEyeToHandTransformed
  The sample now has the following options:
  - eye-in-hand or eye-to-hand
  - transform picking point or entire point cloud
  - renames all the samples to UtilizeHandEyeCalibration

### CODE REVIEW

**Review by chrisasc — COMMENTED** [review]
lgtm, just two small nitpicks

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # C\# samples
 
-This repository contains C++ code samples for Zivid SDK v2.7.0. For
+This repository contains C++ code samples for Zivid SDK v2.6.0. For
```
Comment (SatjaSivcev) [inline README.md:3]:
```suggestion
This repository contains C# code samples for Zivid SDK v2.6.0. For
```

File: source/Applications/Basic/FileFormats/ZDF2PLY/ZDF2PLY.cs [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-Convert point cloud from ZDF a file to a PLY file.
+Convert point cloud from a ZDF a file to a PLY file.
```
Comment (SatjaSivcev) [inline source/Applications/Basic/FileFormats/ZDF2PLY/ZDF2PLY.cs:2]:
```suggestion
Convert point cloud from a ZDF file to a PLY file.
```

[the remark this answers is not in the document]
Comment (SatjaSivcev) [reply source/Applications/Basic/FileFormats/ZDF2PLY/ZDF2PLY.cs]: done

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (SatjaSivcev) [reply README.md]: done

**Review by chrisasc — APPROVED** [review]

**Pull request merged**